src/octopeer-github/content/ContentController.ts

```diff
@@ -105,7 +105,10 @@ class ContentController {
         this.unhookFromDOM();
         this.hookSemanticToDOM(database);
         this.hookTrackersToDOM(database);
-        this.hookMutationObserverToDOM();
+
+        if (Options.get("dataHTML")) {
```

Comment:
I'll merge Mitchell's entire branch in here. Then we'll have to merge his branch first in the end, which shouldn't be a problem I hope.